### DISCUSSION TITLE
Hawaii Hydro Jet Pro 4 cable square pump working 54138

### DISCUSSION THREAD

**Discussion by budgie55** [visualapproach/WiFi-remote-for-Bestway-Lay-Z-SPA#575] [General]
Hi 

Just wanted to say thanks for this superb project, I can confirm per above that the hardware works fine with my pump which is an old Hawaii HydroJet Pro square blue coloured pump with model 54138.

**Comment by visualapproach** [comment]
Thanks!

**Comment by CyberGhostorangefish** [comment]
Hi, do you have any guides on how you made it work with pump model 54138 - thanks

**Comment by budgie55** [comment]
It worked for a little while and then I got the error on the viewing panel. There are notes elsewhere around the error code (sorry can’t recall what it was as was over a year ago), think the solution was certain resistors on the Board, which I’ve not yet had time to try and test. Something I need to get to over next month or so….. sorry I don’t have a guide, all I did was follow the guides and info on here.

Edit: it was the e13 error under (now closed) error 208. Using the additional resistors and capacitors (per the advanced setup in the guide), plus using the red voltage regulator board appears to be the solution. I’ll have to try again and build a new board to be sure myself though for my own spa.

**Reply by georgemckee999** [reply]
Did you get this to work without another e13 error? I have the same pump and get constant e13. I can’t control the pump from the display either when this is connected.
